contents/tutorials/power-users.md

```diff
@@ -0,0 +1,70 @@
+
+As an example, we’ll use `uploaded_file`  as the power user event we care about. We can then either select “completed an event multiple times” or “completed an event regularly.” Because we care about the high volume of usage, and `uploaded_file` isn’t an advanced feature, we’ll choose “completed an event multiple times.” We’ll then set our usage amounts and regularity (which we can tweak later). Once done, it should look something like this:
+
+![Power user cohort](../images/tutorials/power-users/cohort.png)
+
+Make sure to add the title (that is searchable) and description, then select a type (static vs dynamic). You likely want to make this cohort dynamic to ensure it updates with users matching the criteria now. Finally, we’ll click save to use this power user cohort in analysis.
+
+## Analyzing power users
+
+Once we’ve saved a cohort, we immediately get a list of users who fit our power user criteria. We can start by using this list to check for session recordings. To do so, go to recordings, filter by persons and cohorts, then search for your power user cohort. From there you can filter further by events, date, or duration. Analyzing these recordings gives you insights into how power users use your product, and enables you to build a better one for them.
+
+![Session recordings](../images/tutorials/power-users/recordings.png)
+
+You can also use your cohort as a base in trends to do all sorts of analysis. In each of the visualizations (except lifecycle), we can use power users as a filter group. Once the power user filter group is set, we can analyze trends in usage, conversion funnels, retention graphs, user paths, and more.
+
+![Filter group](../images/tutorials/power-users/filter-group.png)
+
+You can also use the power user cohort to compare them against average users. Compare the properties and activity of power users to average users to understand how they differ. For example, we can compare the average file size for power users versus average users, and we might discover average users upload larger files than power users do.
+
+![Power users vs average](../images/tutorials/power-users/average-comparison.png)
+
+With all that, you’ve set up a power user cohort and begun to do analysis. From here, you can tweak the cohort to better fit your definition and tailor your analysis to your product. You can begin to ask questions about your data to help build a better product for your power users.
+
+## Further reading
+
+- Use your newly created power user cohort to calculate session-based metrics like time on site and pages per session with [help from this tutorial](/tutorials/session-metrics).
+- [This tutorial](/tutorials/experiments) teaches you how to run Experiments without feature flags to improve the power user experience.
+- Learn how we make something people want (and create power users by doing it) in [this blog post](/blog/making-something-people-want) by our CEO James.
```

Comment:
I really don't like anchor text like "this tutorial". Not very scanable and terrible for SEO. Always try to link on some meaningful text.

Reply:
Makes sense, changed.